Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml

```diff
@@ -8,7 +8,6 @@
   - type: FlavorProfile
     flavors:
       - meaty
-  - type: Food
```

Comment:
What is this doing here?

Reply:
unused component

Reply:
Please review the [pull request guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). If we find out that these components are actually used, we'd have to revert the whole PR.

Reply:
it inherits basefood which provides foodcomponent
obviously its not used since it doesnt have any fields